evopy/progress_report.py

```diff
@@ -0,0 +1,16 @@
+"""Module containing the ProgressReport class, used to report on the progress of the optimizer."""
+
+
+class ProgressReport:
+    """Class representing a report on an intermediate state of the learning process."""
+
+    def __init__(self, generation, best_genotype, best_fitness):
+        """Initializes the report instance.
+
+        :param generation: number identifying the reported generation
+        :param best_genotype: the genotype of the best individual of that generation
+        :param best_fitness: the fitness of the best individual of that generation
+        """
+        self.generation = generation
+        self.best_genotype = best_genotype
+        self.fitness = best_fitness
```

Comment:
Should be called `self.best_fitness` to be consistent with the `self.best_genotype`

Reply:
Good catch!